src/Card/Header/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Header
+ *
+ * This component is in charge of display
+ * the header of Card component
+ *
+ * @param {string} children - display the title of card component
+ *
+ * @return {jsx}
+ */
+const Header = ({ children, className }) => <div className={className}>{children}</div>;
+
+/**
+ * PropTypes Validation
+ */
+const { node, string } = PropTypes;
+Header.propTypes = {
+  children: node,
+  className: string,
+};
+
+/**
+ * Default props
+ */
+Header.defaultProps = {
+  children: null,
+  className: '',
+};
+
+export default styled(Header)`
+  border-bottom: solid 1px ${({ theme: { palette } }) => palette.lightGrey};
```

Comment:
The color of borders is `veryLightBlue` (cf design). We also need to change that in Sonar